app/src/main/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt

```diff
@@ -24,10 +24,13 @@ class PopularMoviesViewModel @Inject constructor(
         get() = _isLoaderVisible
     val isPlaceholderVisible: LiveData<Boolean>
         get() = _isPlaceholderVisible
+    val isStartingRefresh: LiveData<Boolean>
+        get() = _isStartingRefresh
 
     private val _popularMovies = MutableLiveData<List<Movie>>(emptyList())
     private val _isLoaderVisible = MutableLiveData<Boolean>()
-    private val _isPlaceholderVisible = MutableLiveData(false)
+    private val _isPlaceholderVisible = MutableLiveData<Boolean>()
+    private val _isStartingRefresh = MutableLiveData<Boolean>()
```

Comment:
Couldn't it just be named simpler – `isRefreshing`?

Reply:
Yes, it will be simpler, I changed it 😄